doc/pages/Packaging.md

```diff
@@ -103,7 +103,7 @@ remove: ["ocamlfind" "remove" "project"]
 depends: "ocamlfind"
 ```
 
-The `opam-version`, `maintainer` and `version` fields are mandatory ; you should
+The `opam-version`, `maintainer` and `version` fields are mandatory; you should
```

Comment:
Actually the `version` field should not be specified. You may want to correct this.